src/test/unit/flight_failsafe_unittest.cc

```diff
@@ -226,6 +236,7 @@ TEST(FlightFailsafeTest, TestFailsafeCausesLanding)
 {
     // given
     int callsToMakeToSimulateFiveSeconds = FAILSAFE_UPDATE_HZ * 5;
+    int callsToMakeToSimulateFiftySeconds = FAILSAFE_UPDATE_HZ * 50;
```

Comment:
Again we can see that the FIXME and the use of the counter is having a negative impact even on the tests, the test should be manipulating the response the code under test gets from a call to `micros()` or `millis()`